README.md

```diff
@@ -49,6 +49,10 @@ On Linux, install [Linuxbrew](http://linuxbrew.sh) first.
 * upload the generated `*.tar.gz` file to Google Cloud Storage
 
   ```
+  # If you used --force-core-tap, you need to rename the generated tar so it uses '-' instead of '--'.
+  # Otherwise brew downloading the bottle will not find the correct file.
+  # e.g.: epp--3.1.0.big_sur.bottle.tar.gz to epp-3.1.0.big_sur.bottle.tar.gz
+
```

Comment:
This is a bit of a strange bug when you use that option: https://github.com/Homebrew/brew/issues/4740

I noticed the Catalina version also had the double dashes at https://homebrew-blendle.storage.googleapis.com so I assume that option of --force-core-tap is used before and needed?